servicetalk-loadbalancer-experimental-provider/src/main/java/io/servicetalk/loadbalancer/experimental/DefaultLoadBalancerProviderConfig.java

```diff
@@ -0,0 +1,230 @@
+
+    private final Properties properties;
+
+    private final String rawClientsEnabledFor;
+    private final Set<String> clientsEnabledFor;
+    private final int failedConnectionsThreshold;
+    private final LBPolicy lbPolicy;
+    private final Duration ewmaHalfLife;
+    private final int consecutive5xx;
+    private final Duration interval;
+    private final Duration baseEjectionTime;
+    private final int maxEjectionPercentage;
+    private final int enforcingConsecutive5xx;
+    private final int enforcingSuccessRate;
+    private final int successRateMinimumHosts;
+    private final int successRateRequestVolume;
+    private final int successRateStdevFactor;
+    private final int failurePercentageThreshold;
+    private final int enforcingFailurePercentage;
+    private final int failurePercentageMinimumHosts;
+    private final int failurePercentageRequestVolume;
+    private final Duration maxEjectionTime;
+    private final Duration ejectionTimeJitter;
+
+    private DefaultLoadBalancerProviderConfig() {
+        this(System.getProperties());
+    }
+
+    private DefaultLoadBalancerProviderConfig(Properties properties) {
+        this.properties = requireNonNull(properties, "properties");
+        rawClientsEnabledFor = getString(PROP_CLIENTS_ENABLED_FOR, "").trim();
+        clientsEnabledFor = getClientsEnabledFor(rawClientsEnabledFor);
+        failedConnectionsThreshold = getInt(PROP_FAILED_CONNECTIONS_THRESHOLD, 5 /*ST default*/);
+        lbPolicy = getLBPolicy();
+        ewmaHalfLife = ofMillis(getLong(PROP_EWMA_HALF_LIFE_MS, ofSeconds(10).toMillis()));
+        consecutive5xx = getInt(PROP_CONSECUTIVE_5XX, 5);
+        interval = ofMillis(getLong(PROP_INTERVAL_MS, ofSeconds(10).toMillis()));
+        baseEjectionTime = ofMillis(getLong(PROP_BASE_EJECTION_TIME_MS, ofSeconds(30).toMillis()));
+        maxEjectionPercentage = getInt(PROP_MAX_EJECTION_PERCENT, 20);
+        enforcingConsecutive5xx = getInt(PROP_ENFORCING_CONSECUTIVE_5XX, 100);
+        enforcingSuccessRate = getInt(PROP_ENFORCING_SUCCESS_RATE, 100);
+        successRateMinimumHosts = getInt(PROP_SUCCESS_RATE_MIN_HOSTS, 5);
+        successRateRequestVolume = getInt(PROP_SUCCESS_RATE_REQUEST_VOL, 100);
+        successRateStdevFactor = getInt(PROP_SUCCESS_RATE_STDEV_FACTOR, 1900);
+        failurePercentageThreshold = getInt(PROP_FAILURE_PERCENTAGE_THRESHOLD, 85);
+        enforcingFailurePercentage = getInt(PROP_ENFORCING_FAILURE_PERCENTAGE, 0);
+        failurePercentageMinimumHosts = getInt(PROP_FAILURE_PERCENTAGE_MIN_HOSTS, 5);
+        failurePercentageRequestVolume = getInt(PROP_FAILURE_PERCENTAGE_REQUEST_VOL, 50);
+        maxEjectionTime = ofMillis(getLong(PROP_MAX_EJECTION_TIME_MS, ofSeconds(90).toMillis()));
+        ejectionTimeJitter = ofMillis(getLong(PROP_EJECTION_TIME_JITTER_MS, ZERO.toMillis()));
+    }
+
+    private LBPolicy getLBPolicy() {
+        return getString(PROP_LOAD_BALANCING_POLICY, "p2c")
```

Comment:
So it defaults to p2c but if something else is specified it goes to round robin - if something unsupported is specified, should we fail (or at least log?)